SPECS/openssl/openssl.spec

```diff
@@ -62,6 +62,7 @@ Patch52:  0052-Allow-SHA1-in-seclevel-1-if-rh-allow-sha1-signatures.patch
 # # See notes in the patch for details, but this patch will not be needed if
 # # the openssl issue https://github.com/openssl/openssl/issues/7048 is ever implemented and released.
 Patch80:  0001-Replacing-deprecated-functions-with-NULL-or-highest.patch
+Patch81:  CVE-2024-5535.patch
```

Comment:
According to the [security advisory](https://www.openssl.org/news/secadv/20240627.txt), this is fixed in 3.3.2, so a comment here saying to remove this patch if we ever upgrade would be helpful.

```suggestion
# Remove if we upgrade to 3.3.2 to or later. https://www.openssl.org/news/secadv/20240627.txt
Patch81:  CVE-2024-5535.patch
```